moulinette/utils/filesystem.py

```diff
@@ -290,7 +288,9 @@ def mkdir(path, mode=0o777, parents=False, uid=None, gid=None, force=False):
 
     # Create directory and set permissions
     try:
+        oldmask = os.umask(000)
```

Comment:
I'm not sure about that

Reply:
Eh what is it supposed to do ? o.O

Reply:
without this "fix", "tweak", whatever... the `mode` (permission) of the folder may not be the same as the one given in argument. (depending on current value of the [`umask`](https://wiki.archlinux.org/index.php/Umask)).

But I'm not 100% sure if we want it... I can remove and adapt the tests accordingly

Reply:
Uuuuh wokay ... Not sure to understand but let's roll with it

Reply:
You can find answers here I guess: https://stackoverflow.com/questions/5231901/permission-problems-when-creating-a-dir-with-os-makedirs-in-python